orders/models.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+
+from sqlalchemy import (
+    DECIMAL, Column, DateTime, ForeignKey, Integer,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+
+class Base(object):
+    created_at = Column(
+        DateTime,
+        default=datetime.datetime.utcnow,
+        nullable=False
+    )
+    updated_at = Column(
+        DateTime,
+        default=datetime.datetime.utcnow,
+        onupdate=datetime.datetime.utcnow,
+        nullable=False
+    )
+    deleted_at = Column(DateTime, nullable=True)
+
+    def delete(self):
+        if self.deleted_at is None:
+            self.deleted_at = datetime.datetime.utcnow()
```

Comment:
I wonder if using our `Base` is too much for this example. It's a nice pattern but may be confusing since it's non-essential.

Maybe we can keep it but remove the "soft-delete"?